libopenage/CMakeLists.txt

```diff
@@ -127,8 +144,9 @@ endif()
 
 get_config_option_string()
 
-configure_file(config.h.in ${CMAKE_CURRENT_SOURCE_DIR}/config.h)
-configure_file(config.cpp.in ${CMAKE_CURRENT_SOURCE_DIR}/config.cpp)
+configure_file(config.h.in config.h)
+configure_file(config.cpp.in config.cpp)
+include_directories(${CMAKE_BINARY_DIR})
```

Comment:
this include does a lot of magic. would be cool if you commented a bit about it.